README.adoc

```diff
@@ -320,7 +320,8 @@ The actuator exposes the following:
 
 NOTE: There is also an `/actuator/shutdown` endpoint, but, by default, it is visible only
 through JMX. To http://docs.spring.io/spring-boot/docs/{spring_boot_version}/reference/htmlsingle/#production-ready-endpoints-enabling-endpoints[enable it as an HTTP endpoint], add
-`management.endpoints.web.exposure.include=health,info,shutdown` to your `application.properties` file.
+to your `application.properties` file `management.endpoint.shutdown.enabled=true`
```

Comment:
Change to "`management.endpoint.shutdown.enabled=true` to your `application.properties` file"

Reply:
agreed